specs/posix/process_events.table

```diff
@@ -4,8 +4,10 @@ schema([
     Column("pid", BIGINT, "Process (or thread) ID"),
     Column("path", TEXT, "Path of executed file"),
     Column("mode", BIGINT, "File mode permissions"),
-    Column("cmdline", TEXT, "Command line arguments (argv)"),
-    Column("cmdline_size", BIGINT, "Actual size (bytes) of command line arguments"),
+    Column("cmdline", TEXT, "Command line arguments (argv)",
+        aliases=["argv", "args"]),
```

Comment:
I'd recommend against using aliases unless you're trying to maintain support for deprecated or renamed columns. These will cause a significant amount more copying for one of the largest string values in the table schema. It will also complicated the documentation/schema if you're inspecting `.schema`.